src/util/records.py

```diff
@@ -830,6 +832,10 @@ def add_ref(ref):
                         for ref_match in re.finditer(r"\bref\((['\"])(.*?)\1\)", eval_node.get("eval")):
                             add_ref(ref_match.group(2))
 
+    if not found:
+        suffix = " in %r module" % from_module if from_module != module else ""
+        raise SleepyDeveloperError("Cannot find %r%s" % (xmlid, suffix))
```

Comment:
It could be a rugpull[^1] in standard, let's not blame always the dev :D
```suggestion
        raise ValueError("Cannot find %r%s" % (xmlid, suffix))
```
[^1]: I agree, it's not OK to remove an xmlid in standard but ...